ngraph/python/tox.ini

```diff
@@ -23,8 +23,8 @@ commands=
   flake8 {posargs:src/ setup.py}
   flake8 --ignore=D100,D101,D102,D103,D104,D105,D107,W503 tests/  # ignore lack of docs in tests
   mypy --config-file=tox.ini {posargs:src/}
-  pytest --backend={env:NGRAPH_BACKEND} tests/test_ngraph/test_core.py -v
-  pytest --backend={env:NGRAPH_BACKEND} tests/test_onnx -v -k 'not test_zoo_models.py'
+  pytest --backend={env:NGRAPH_BACKEND} tests/test_ngraph tests/test_onnx -v -k 'not test_zoo_models.py'
```

Comment:
Why is `test_zoo_models` still excluded?

Reply:
Because models are not ready to merge yet - they will be added with the next PR with additional models

Reply:
Finally, zoo models cant be added in this PR